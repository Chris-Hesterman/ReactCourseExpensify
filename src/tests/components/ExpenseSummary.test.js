import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';

test('should correctly render info for no expenses', () => {
    const wrapper = shallow(<ExpenseSummary expensesCount={0} expensesTotal={0} />);

    expect(wrapper).toMatchSnapshot();
});

test('should correctly render info for 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expensesCount={1} expensesTotal={345} />);

    expect(wrapper).toMatchSnapshot();
});

test('should render correctly for 2 expenses', () => {
    const wrapper = shallow(<ExpenseSummary expensesCount={2} expensesTotal={234234} />);

    expect(wrapper).toMatchSnapshot();
});