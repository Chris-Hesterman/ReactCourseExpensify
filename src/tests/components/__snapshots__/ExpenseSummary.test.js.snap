// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render info for 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     expense 
    totalling 
    $3.45
  </h1>
</div>
`;

exports[`should correctly render info for no expenses 1`] = `
<div>
  <h1>
    Viewing 
    0
     expenses 
    totalling 
    $0.00
  </h1>
</div>
`;

exports[`should render correctly for 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     expenses 
    totalling 
    $2,342.34
  </h1>
</div>
`;
